pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1319,6 +1319,13 @@ func (m *kubeGenericRuntimeManager) SyncPod(ctx context.Context, pod *v1.Pod, po
 	if isInPlacePodVerticalScalingAllowed(pod) {
 		if len(podContainerChanges.ContainersToUpdate) > 0 || podContainerChanges.UpdatePodResources {
 			m.doPodResizeAction(pod, podStatus, podContainerChanges, result)
+			if result.SyncError != nil {
```

Comment:
The error here is incomplete. Not only is there an error here, but there are also Resize errors in other places.

Reply:
This PR only handles resizing errors that occur within the runtime, such as in #124786. I'm happy to close this PR in favor of yours if it is cleaner for review @hshiina

Reply:
If you don't mind, please close this PR.